custom_components/homewhiz/appliance_controls.py

```diff
@@ -307,6 +344,35 @@ def build_controls_from_progress_variables(
     return result
 
 
+def build_control_from_remote_control(remote_control: Optional[ApplianceRemoteControl]):
```

Comment:
Maybe add return type since mypy is now part of pre-commit?

Reply:
Done. added https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_untyped_defs